src/networking.c

```diff
@@ -405,6 +405,22 @@ void addReplyErrorLength(client *c, const char *s, size_t len) {
 
 /* Do some actions after an error reply was sent (Log if needed, updates stats, etc.) */
 void afterErrorReply(client *c, const char *s, size_t len) {
+    /* Update per command error stats */
+    if (c->cmd) {
+        c->cmd->failed_calls++;
+    }
+    /* Increment the error stats
+     * If the string already starts with "-..." then the error prefix
+     * is provided by the caller. Otherwise we use "-ERR". */
+    if (s[0] != '-') incrementError("ERR", 3);
+    else {
```

Comment:
as much as i hate wasting lines on braces, this seems to be the style in redis
```suggestion
    if (s[0] != '-') {
        incrementError("ERR", 3);
    } else {
```